packages/desktop/views/dashboard/collectibles/views/CollectiblesGalleryView.svelte

```diff
@@ -4,6 +4,22 @@
     import { localize } from '@core/i18n'
     import { nftSearchTerm, queriedNfts, ownedNfts } from '@core/nfts'
     import { FontWeight, Illustration, Text, SearchInput, NftGalleryItem, Button, ButtonSize } from 'shared/components'
+    import VirtualList from '@sveltejs/svelte-virtual-list'
+    import { onMount } from 'svelte'
+
+    let windowHeight = document.body.clientWidth
```

Comment:
you can use `svelte:window` for this I think and it will make the code a bit cleaner